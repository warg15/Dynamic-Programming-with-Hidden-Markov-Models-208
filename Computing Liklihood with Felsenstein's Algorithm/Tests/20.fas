>Rhinoceros-hornbill
CCGTCCCTCGGGAAAGAAGACTGAACGATGTGTTCCTTGGTATTCTACTGGGGGAAAATAACAGGCAACCTATTTGTTGCCTGCGTCTGGGAAAAGCGGGAGCCATGGACATAGTGGCGCAAGAAACAAAACGAGTTGTCTTTGGCAAGTGTTTTCTCTAATCTCGTAACGGAAGGCAGGCAAACTGGAAGAATAGTTCT
>Emperor-penguin
CTGCCCCTCGGGAAAGAGGACTAGACCAAGTGATATTCAATATTCTAGTGGGGGGAAAGGCCGGGCAACCTATTTGTTGTCTGCGTAAGTGAAAGGCGGGAGCCATGGACATAGTGACGCAAGAAACAGAACGACTTGTTCTGAGCACGTGTTTTCTCTAATCTCGTAACAGAGGGCAGGCAAACTGTAAGAATAGTCGA
>White-tailed-tropicbird
CTGCCCCACGGGAAAAAGGACTAGACCGAGTGATATTCAATATTCTAGTGGGGGGAAAGGCCGAGCAACCTATTTGTTGCCTGCGCCAGGGAAGGGCGGGAGCCATGGACATAGTGACGCAAGAAGCAGAACGACTTGTTCTGGGCACGTGTCTTCTATAATCTCGTAAAAGAGGGCAGGCCAACTGGAAGAATAATTGA
>American-flamingo
CTGCCCCCCGGGAAAGGGGACTAGAACAAGTGATATTCAATATTCTAGTGGGGGGAAAGGTCGGGCAACCTATTTGTTGCCTGCGTCAGGGAAAGGCGGGAGCCATGGAGATAGTGACGCAAGAAGCAGAACGACTTGTTCTGAGCACGTGTTTTCTCTAATCTCGTAACAGAGGGCAGGCAAACTGGAAGAATAGTTGA
>Grey-crowned-crane
CTATCCCTCGGAAAAGAAGACTAGACCAAGTGATATTCAATATTCTAGTGGGGGGAAAGGCCGGCCAAACTATTTGTTGCCTGCGTCAGGGAAAGGCGGGACCCATGGACATAGTAAAGCAAGAAACAGAACGACTTGTTCTGATCACGTGTTTTCTCTAAGCTCGTAACAGAGGGCAGATAGACTGGAAGAATAATCGA
>Bar-tailed-trogon
CTGCCCCTCAGGAAAGAGGACTAGACGAAGTGTTATTCAATACTCTAGTGGGGGGAGACGCCGGGCAACCTATTTGCTGCCTGCGCCAGCAAAAGGCGGGGGCCATGGCCATAGTGACGCAATAAGCAGAACCACCTGTTCTGAGCACGTGTTTTCTCTAATCTCGTAACAGAGGGCATACAAACTAGAAGAGTAGATGA
>Zebra-finch
TCGCTCCTCGGGAAAGAAGACCAGACAAAGTGTTATTTAATATCCTAGTGGGAGGAAACATCCGGTAACCTATTAATTGCATCCGTCAGAGATAGGCGGGAGCAACGAACACAACGCTGCAAGAAACAGAACGATTCGGTCTTAGCATGTGTTTTCTTTAATCTCGTGACAAAGGGTAGGCAAAATGATAGAATAGTTTG
>Chuck-wills-widow
CTGCCACTCGGAAAAGGAGACTAGACCAAGTAGTATTCAATATTCTAGTGGGCCGAAAGGCCGGGCAACCTATTTGTTGCCTGCGTCAGGAATAGGCGGGCGCCATTGACATAGTGACGCAAGAAACAGAACGACTCGCTCTGAGCAAGTGTTTTCTCTAATCTCGTAACAGAGGGTAAGCAGACTGGGAGTATAGTTGA
>Speckled-mousebird
CTGCCCCTCTGGAAAAAAGACTAGACAAAGTCTTATTCAATATTTTATTGGGGGGAAATGCCGAGCAACCTACTTGTTGCCTGCGAAAGGAAAACGCAGGAGCCAGGGACATAGTGACGCAGGAAACAGAAGAACTTGTTCTGAGCACGCGTTTTTTCTAACCTCGTAACAGAGGGCAGGCAAACTGGAAGAATAGTTGA
>Little-egret
CTGCCCCGCGGGAAAGAGGACTAGACCAAGTGATATTCAATATTCTAGTGGGGGGAAAGGCCGGGCAACCTATTTGTCGCCTGCGTCAGGGAAAGGCGCGAGCCATGGACATAGTGACGTAGGAAACAGAACGACTTGTTCTGAGCACGTGTTTTCTCTAATCTCGTAACAGAGACCAGGCAAACTGGAAGAATAGTTGA
>Rifleman
ACGCTTCTCGGGAAAGAGGATTAAACAAAGTGTTATTTCATATTCTAGTGGGAGGAAATGCCGGGCAACCTATTTGTTGCCTACGTCAGGGAAAGTCTGGAGACACGAACATAATGACGCAAGAAACAGAACGATTTGTTCTTAGCATGTATTCTCTCTAATCTCGTGACAGAGGCCAGGCAAACTGAAAGAATAGTTTA
>Annas-hummingbird
CTGACCCTCGGGAAAGAGGGCTAGACCAAGTGAGATTCAATATTCTAGGGGTGGGACAGGCCGGGCAACTTATTTGTTGCCTGTGTCAGGGAAAGGCGGGAGCCATGGATATATTGACGCAAAATACAGAATTACTTGTTCTGAGCAAGTGTTTTCTCTAATCTCGTAACAGAGGGCATAAATATTGGAAGAGTAGTTGA
>Kea
CTGCCCCTTGGGAAAGAGGACTACGCCAAGTGTTATTCAATATTCTACTGGAAGGAAATGCCGGGCAACCTATTTGTTGCCTGCGTCGGGGAAAGGCGGGAGCCATGCACATAGTGACGAAAGGAACAGAACGAATTGTTCTGAGCACGTGTTTTCTCTAATCTCGTAACAGAGGGTAGGCGGACTGGAAGAATAGTTTA
>Barn-owl
CTGCCTCTAAGGAAATAGGATTACACGAAGTGTTATTCAATATTCTAGTGGGGGGAAATGCCAGGCAACCTATTTGTTGCCTGCGTCAGGGAAAGGCGGGAGCCATGAACATAGGCACGCAAGAAACAGAACGACTTGTTCTGAGCACTTATTTTCCCTAACCACGCAACAGAGGGCAGGCAAACTGGAAGAATAGTTGA
>Turkey-vulture
CTGCCCCTCGGGAAAGAGAATTAGACCAAGTGTTATTCAATATGCTAGTGGGGGGAAATGCCGGGCAACCTATTTGTTGCCTGCGTCAGGGAAAGGCGGGAGGCATGGACATAGTGACGCAAGAAACAGAACGACTTGTTCTGAGCACGTGTTTTCTCTAATCTCGTAACAGAGGGCAGGCAAACTGGAAGAATAGCTGA
>MacQueens-bustard
CTGCCCCTCGAGAAAGAGGACTAGACCAATTGATACTCGATATTCTAGTGGGGGGAAAGGCCGGGCAACCTATTTGTTGCCTGCGTCAGGGAGAGGCGGGAGCCATGGACATAGTGACGCAAGAAACAGAACAACTCGTTCTGAGCATGTGTTTTCTCTAATATCGTAACAGAGGGCAGGCAAACTGGAAGAATAGTTGA
>Red-legged-seriema
CTGCCCCTCGGGAAAGAGGACTAGACCAAGCGTTATTCAATATTCTAGTGGGAGGAAATGCCGGACAACCTATCTCTTGCCTGCGTCAGGGAAAGGCGGGAGTCATGGACCTAGTGACGCAAGAAACAGAACGACTTGTTCTGAGCACGTGTTTTCTCTAATCTCGTAACAGAGGGGAGGCAAACTGGAAGAATAGTTGA
>White-tailed-eagle
CTGCCCCCCGGGAAAGAGGACTAGATCAAGTGTTATTCAATATTCTAGTGGGGGGAAATGCCGAGCAACCTATTTGTTGCCTGCGTCAGAGAAAGGCGTGAGCCATGAACATAGTGACGCAAGAAACAGAACGACTTGTTCTGAGCATGTGTTTTCTCTAATCTCGTAACAGAGGGCAGGCAAACTGGAAGAATAGTTGA
>Yellow-throated-sandgrouse
CTGCTTCTCGGGGAAGAGCACTAGACCAGGTGATACTCAATGTTCTAGTGGGGGGAAAGGCCGGGCAACCTATTTGTTGCGTGCGTTAGGGAAAGGCAGGAGCCATGGACATAGTGACGAAAGAAACAGAACGGCTTGTTGTGAGCTCGTGTTTTCTCTTATCTCGTAACAGGGGATGGGCAAATTGCAAGAATAGTTCC
>Chimney-swift
CTGCCCCTCGGGAAAGAGGATAAGACCAAGTGATATTCAATACTCTAGTGGGGGGACAGGCCGAGCAACCCATCTGTTGTCGGCGTTAGGGAAAGGCGGGAGCCATGGATATAGTGACGAAAGAAACAGAACTACTTGTTCTGAGCGAGTGTGCTCTCTAATTTCGTAACAGAAGGGATGCAAACTGGTAGAGTAGTTGA
>Adelie-penguin
CTGCCCCTCGGTAAAGAGGACTAGACCAAGTGATATTCAATATTCTAGTGGGGGGAAAGGCCGGGCAACCTATTTGTTGCCTGCGTAAGTGAAAGGCGGGAGCCATGGACATAGTGACGCAAGAAACAGAACGACTTGATCTGAGCACGTGTTTTCTCTAATCTCGTAACAGAGGGCAGGCAAACTGGAAGAATAGTTGA
>Red-throated-loon
CTGCCCCTCGGGAAAGAGGACTAGACCAAGTGATATTCAATATTCTAGTGGGGGGGAAGGACGGGCAACCTATTTGTTGCCTGCGTCAGCGAAAGGCGGGAGCCATGGATATAGTGACGCAAGAAACAGAACGACTTGTTCTGAGCACGTGTTTTCTCTAATCTCGTAACAGAAGGCAGGCAAACTGGAAGAATAGTTGA
>Pekin-duck
CTGCTTCTCGGGAAAGAGGACTAGACTAAGTGATGTTCAATATTCGCCTGGGGGGAAAGGTTGGGCAACCTACTTGGTGCTTGCGTAGGGGAAAGAGGGGAGCCATTGACATAGTGACACAAGAAACAGTACGACTTGTTTTGAGGACGCGTCTTCTCTAATCTCGTAACTTAGGGCAGGCGAACCGAAAAAATAGTTGA
>Budgerigar
CTGCCCCTCGGGAAAGAGGTCTACACGCAGTGTTATTTAATGTTCTATTGGGAGGAAATGGCGGGCAACCTATTTGTTACCTGAGTCAGGGAAAGGCGGGAGCCATGGATATAGTGACGCATGGAACAGAACGACTTGTTCTGAGAACGTGTTTTCTCTAATCTCGTAACAGAGGGTAGGCAGACTGGAAGAATAGTTTA
>Turkey
CTGCCCCACGGGAAGGAGTACTAGAACAAGTAATGTTCAATATTCTACCGGGGGAGGGGGACGAGGAACCTATTTCTTACCTGCGTAAGTGAAAGAAGCGAACCATGGTCACATTGCCGCAAGAAACAATACAACTTGTTTTAAGGACGTGCCTTCTCTAATCTCGTAACATAGAGAAGACTAACCGGAAGGATACCTGA
>Downy-woodpecker
CTGCCCCTGGGGAAGGAAAACCGGACGGTGTGCTACTCAACATTCTAGTGGGGGGAAATACCGGACAACCTATTAGTTGCCTGCTTCGGGGGAAGGCGGGAGCTATGGACGTAATAACGCAAGGAACAGTAAGACTTGTTTTGAGCGCGTATTTTCTCTAATCTCGTAAGAGAGGGCAGGCAAAATTGAAGAATAGTTGA
>Dalmatian-pelican
CTGCCCCTCGGGAAAGAGGACTAGACCAAGTGGTATTCAATATTCTAGTGGGGGGAACGGCTGGGCAACCTATTTGTTGCCTACGTCAGGGAAGGGTGGGAGCCATGGACATAGTGACGCAACAAACAGAACGACTTGTTCTGAGCATGTGTTTTCTCTAATGTCGTAACAGAGGACAGGTAAACTGGAAGAATAGTTGA
>Golden-collared-manakin
TCGCTCCTCGGGAAAGAGGACTAGACCAAGTGTTGTTTAATATTCTAGTGGGAGGAACTGTCGGGTAACCTATTTGTTGCCTGCGGCAGGGAAAGACGAGAGCCACGAATACAATGACGCTAGAAACAGAACGATTTGTTCTTAGCATGTGTTTGCTCTAATCTCGTGACAGAGAGCAGGCAAAGTAGAAGAATAGTTTA
>Northern-fulmar
CTGCCCCTCGGGAAAGAGGACTAGACCGAGTGATATTCAATATTCTAGTGGGGGGAAAGGCCGGGCAACCTATTTGTTGCCTGCGTCAGGGAAAGGCGGGAGCCATGGACGTAGTGACGCAAGAAATAGAACGACTTGTTCTGAGCACGTATTTTCTCTAATCTCGTAACAGAGGGCAGGCAAACTGGAAGGATAGTCAA
>Medium-ground-finch
TCGCTCCTCGGGAAAGAGGACCAGACAAAGTGTTACTTAATATCCTAGTGGGATGAAATATCCAGTAACCTCTTTATTGCATGCGTCAGAGAAAGGCGGGAGAAACGAACACAATGCTGTAAGAAACAGAACGATTTGGTCTTAGCATGTGTTTTCTCCAATCTCGCGACAGAGGGTAGGCAAAATGGTAGAATAGTTTG
>Great-crested-grebe
CTGCCCCTCGGGAAAGAGGACTAGCACAAGTGATATCCAATGTTCTAGTGTGGGGAAAGGCTGGGCAACTTATTTGTTGCCTGCGTCAGGGAATGGCGGGAGCCAAGGACATAGTGACGTAAGAAACAAAACGACTTGTTCTGAGCACGTATTTTCTCTAATCTCGTAACAGAGGGCGTGCAAACTGGAAGAATAGTTGA
>Cuckoo-roller
CTGCCCCTCGGGAAAGAGGACTAATCCAAGTGTTATTCAATATTCTAGTGGGGGGAAATGCCAGGCAACCTATTTGTTGCCTGCGTCAGGGAAAGGCGTGAGCCATGGACATAGTGAAGCAAGAAAGAGAACGACTTGTTCTAAGCACGTGTTTTCTCTAATCCCGTAACAGAGGGCAGGCAAACTGGAAGAATAGTTGA
>Sunbittern
CTGCCCCTCGGGAAGGAGGACTAGACCAATTGATATTTAATATTCTAGTGGCTGAAAAGGTCGAGCAACCTATTTGTTGCCTGCGACAGGGGTAGGCGGGAGCCATGGATATAGTGACGTAAGAAACAGAACGGCTTGTTCGGAGCGCGGGTTCTCTCTAATCTCGTAACAGAAGGCAGGCAAAATGGAAGAATAGTTGA
>Peregrine-falcon
CTGCCCCTCGGGAAAGAGGACTAGACCAAATGTTATTCAATATTCTAATGGGAGGAAATGCCGGGCAACTTATTTGCTGCCTGCGTCAGGGAAAGGGGGCAACCACGGACATAGTGACGCAGAAAACAGAACGAGTTGTTCTGAGCACGTGATTTCCCTAATCTCATAACAGAGGGGAAACAAGCTGGAAGAATAGTTGA
>Carmine-bee-eater
ATGCCTCTTGAGAAAGAAGACTAGAGGAAGTGTTATTCAAGGTCCTAGTGGGGGGAAATTATGGACTACCTATTTGTTCCCTGAGTTCACCACAGGCGGGAGCCATGGACATAGTGACGCAAAAAAAAGAACGACTTGTTCTGAGCACGTGTTTTCTCTAATCTCATAACAGAGGGCAGGTAAACTGGAAAAATAGTTGA
>Crested-ibis
CTGCCCCTCGGGAAAGAGGACTATACCAACTGATATTCAATATTCTAGTGGGGGGAAAAGCTGGGCAACCTATTTGTTGCCTGCGTCAGGGAAAGGCGGGAGCCATGGACATAGTGACGCAAGAAACAGAACGACTTGTTCTGAGCACGTGTTTTCTCTAATCTCGTAACAGAGGGCAGGCAAACTGGAAGAATATTTGA
>Common-cuckoo
CTGCCCCTCGGGAAAGAGGACTAAGCCACATGATATTCAGTATTCTAGTGGAGGGAAAGGCTGGGCAACCTATTTGTAGCCTACGTCCGGGAAAGGCGGGAGCCATGGACATAGTGACGCAAGAAACAGAACGACTTCTTCTGAGCACAAGTTTTCTCTAATCTCGTAATAGAGGGCAGACAGACTGGAAGAATATTTTA
>American-crow
TCGCTCCTCGGGAAAGAGCACCAGACCAAGTGTTATTTAATATTCTAGTGGGGGGAAAGGTCCGGTAACCTATTTATTACATGCGTCAGAGAAAGGCGGGAGCAACGAACACAGTGCTGCAAGAAACAGAACGATTTGTTCTTAGCATGTGTTTTCTCTAATCTCGTGACAGAGGATAGGCAAAATGGCAGAATAGTTTG
>Pigeon
CTGCCCCTCGGAAAAGAGGACTAGACCAATGGATATTTAATATCCCAGCGGGGGAAAAGGCCGGGCAACCCGTGTTTTGCCTGCGTCAGGGAAAGCCGGGCTCCATGACCATAGTGACGCAAGAGACAGAACAACTTGTTCTGAGCACGTGTTTTCTCTAATCACGTAACAGGGGGCAGGCAAATCGGAAGAATAGTTGA
>Brown-mesite
CTGCCCCTCGGGAAAGAGGACTAGACCAAGTGTTATTCAATATTCTAATGGGGGAAAAGGCCGGGCAACGTATTTGTTGCCTTCGTCATGGAAAGGCGAGAGACATAGACATAGTGACGCAAGAAATAGAACGACTTGTTCTGAGCACGTGTTTTCTCTAATCTCGCAACAGGGGGCAGGCAAACCGGAAGAATAGTCAA
>Great-cormorant
CTGCACCTCGGGAAAGACGACTAGACCAAGTGAGATTCAATATTCTAGTGGGGGGAAAGGCCGGGCAACCTATTTGTTGTCTGCGTCAGGGAAAGGCGGGCGCCATGGACATAGCGATGCAAGAAACAGGACGACTTGTTCTAAGCACGTGTCTTCTCTAATCTCGTAACATAGGACAGGCAAACTGGAAGAACAGTCGA
>Red-crested-turaco
CCGCCCCCCGGGGAAGAGGACTAGACCAAGTGATATTCAATATTCTAGTGGGGGGAAAGGCCGGGCAACCTATTTGTTGCCTGCGTCAGGGAAAGACGGGAGCCATAGGCATAGTGACGCAGGGAGTAGAACGACTTGTTCTGAGCATGTGTTTTCTCTAATCTCGCAACCGACGGCAAGCAAACCGAAAGCATAGTTGA
>Chicken
CTGCCCCTCGGGAAGAAGGATTAGACCATGTGTTGTTCAATATTCTATCGGGGGAGGAGGACGAGGAACCTATTTGTTACCTGCGTATGGGAAAGGAGGGTACCATGGTCACATTGCCGCAAGAAACAATACAACTTGTTTTAAGGACGTGCCTTCTCTAATCTCGTAACATAGAAAAGCCTAACGGGAAGGATAGCGAA
>Common-ostrich
CTGCCCCACGGGAAAGAAGACTAGACCAAGTGATATTCAATATTTTACTGGAGGGGAAGGTCGAGCAACCTATTTGTTGTCTGCGTCAGGGGAAGGCGGGAACCATGGACATAGTGACGTAAGAAACAGTACGACTTGTTCTGGGTACGTGTTTTTTCTAACCTCTCAACAGACGGCAGACATACTGGGAAAATAGTTGA
>Killdeer
CTGCCCCTCGGGAAAGAGGATTAGACCAAGTGATATTCAATATTCTAAAGGGGAGCAAGGCCGGGCAACCTCTTAGTTGTCTGCGTCAAGGAAAGGCGGGAGCCATGGACATAGTGACGCAAGAAACAGAATGACTTGTTCTGATCACGTGTTTTCTCCAATCTCGTAACAGAGGACAGGCAAACTGGAAGAATAGGTGA
>Hoatzin
CTACCCCTCGAGAGAGAGGACTAGACCATGTGATATTTACTATTCTAGCGGGGGCAAAGGCCGGACAACCTATTTGTTGCCTGGGTCAGGGAAAGGCGGTAGCCATGGACATAGTGACGCAAGAAACCGTACGACTTGTTCTGAACACGTGTATTCTCTAATCTCGTAATAGAGGACAGGCAAACTGGAAGAATAAGTGA
>Bald-eagle
CTGCCCCCCGGGAAAGAGGACTAGATCAAGTGTTATTCAATATTCTAGTGGGGGGAAATGCCGAGCAACCTATTTGTTGCCTGCGTCAGAGAAAGGCGTGAGCCATGAACATAGTGACGCAAGAAACAGAACGACTTGTTCTGAGCATGTGTTTTCTCTAATCTCGTAACAGAGGGCAGGCAAACTGGAAGAATAGTTGA
>White-throated-tinamou
CTGCCTCAGGTGAAGCCAGACTAGGCCAGTTGACATTTGATGTTTTACTGGGGGGGAAGGTCGGGCAACCTATTTGTTGTCTGCGTCAGGGAAAGGCGGGAAGCATGAACATGGTGACGTAGGAAACAGCACGACTTGTACTGGGGACGTGTTTTTTCTAACCTCGTAACAGTCGGCAGACATAATGGGAATGTAGTTGA
