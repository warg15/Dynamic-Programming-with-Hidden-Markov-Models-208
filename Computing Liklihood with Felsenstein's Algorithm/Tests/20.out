-3616.315106119392
